src/validation_workflow/validation.py

```diff
@@ -0,0 +1,24 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+
+from abc import ABC, abstractmethod
+
+
+class Validation(ABC):
+    base_url = "https://artifacts.opensearch.org/releases/bundle/"
+
+    def __init__(self, version: str, distribution: str, platform: str, projects: list) -> None:
+        self.version = version
+        self.distribution = distribution
+        self.platform = platform
+        self. projects = projects
+
+    @classmethod
```

Comment:
Why is this a classmethod after all?

Reply:
Removing class method threw me an error `Missing positional argument "self" in call to "download_artifacts" of "Validation"` so I let it stay Peter.